schedule/yast/yast_hostname/yast_hostname+dhcp_hostname_x86_64.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: yast_hostname+dhcp_hostname
+description: >
+  Test suite uses default option to set dhcp (using ifgf=*=dhcp), hostname.
+  Test validates if installation can successfully start in case of usage of
+  these parameters.
+vars:
+  EXIT_AFTER_START_INSTALL: '1'
```

Comment:
this is not needed with yaml schedule. Please correct in the other *hostname* test suites.

Reply:
removed.